Source/WebCore/html/canvas/CanvasRenderingContext.h

```diff
@@ -79,10 +79,17 @@ class CanvasRenderingContext : public ScriptWrappable, public CanMakeWeakPtr<Can
 
     virtual void clearAccumulatedDirtyRect() { }
 
-    // Called before paintRenderingResultsToCanvas if paintRenderingResultsToCanvas is
-    // used for compositing purposes.
-    virtual void prepareForDisplayWithPaint() { }
-    virtual void paintRenderingResultsToCanvas() { }
+    // Canvas 2DContext drawing buffer is the same as display buffer.
+    // WebGL, WebGPU draws to drawing buffer and composites to display buffer.
```

Comment:
Nits: Is the display buffer composited to, or is it the source that is composited? Is something like:
```
// WebGL, WebGPU draws to the drawing buffer and the display buffer is sent to the compositor.
```
???

Reply:
Thanks!